### ISSUE TITLE
Powershell: enclosed subexpression in string breaks color matching

### ISSUE THREAD

**Issue by moocowcat** [CoenraadS/BracketPair#94]
Color matching is broken when a string contains a subexpression enclosed by any type of bracket.

![image](https://user-images.githubusercontent.com/37227202/37428140-90ad0190-2788-11e8-84c6-ebcd84f0c23b.png)

**Comment by CoenraadS** [comment]
Do the square brackets mean anything or are they part of the string?

**Comment by moocowcat** [comment]
Part of the string.  In this scenario they are used to call attention to specific parameters or returned values in output. 

Example: 
Found Azure endpoint [database.windows.net] for SQL Azure logical servers.

**Comment by CoenraadS** [comment]
Should be fixed in v1.0.21

**Comment by flolilo** [comment]
Not sure if this still is the same error - if not, I can open up a new issue.

![grafik](https://user-images.githubusercontent.com/30194876/38966250-930c48be-4381-11e8-8d66-568cfd2a7bdb.png)

Code (as seen in the screenshot):

```Powershell
While($true){
    Write-Output "$(($InFiles | Where-Object {$_.ToCopy -eq 0}).count)"
}
```

VS Code's `editor.matchBrackets` properly highlight the outer curly brackets, but Bracket Pair links the first open-parenthese of Write-Output to the close-curly bracket.

If I remove the inner parenthese(s), Bracket Pair works properly, again.

**Comment by CoenraadS** [comment]
@flolilo Tracking here: https://github.com/PrismJS/prism/issues/1407

**Comment by flolilo** [comment]
@CoenraadS as far as I see, the issue should be fixed in prismJS, right?

I hope you do not mind me asking: Why is it not fixed in Bracket Pair (as of version 1.0.53)?

I know that developing is a very time-consuming process and that this is probably not the most relevant issue for most of the people - so please do not think of this question as "Come on, man, fix it! What am I not paying you for!?", but rather as a "Is there anything that I can do?"
